Snowflake.Data.Tests/SFBaseTest.cs

```diff
@@ -30,8 +30,19 @@ public SFBaseTest()
         [OneTimeSetUp]
         public void SFTestSetup()
         {
-            string testConfigString = Encoding.UTF8.GetString(Properties.Resources.parameters);
+#if NET46
+            log4net.GlobalContext.Properties["framework"] = "net46";
+            log4net.Config.XmlConfigurator.Configure();
 
+#else

```

Comment:
So this means two test binaries is built? one for .Net framework and other for .Netcore? If so, are you running tests for both binaries?

Reply:
Yes. Running tests for both.